config/Shared/config_local.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/**
+ * This configuration is used for TESTING only and will never be used in production!
+ */
+
+use Spryker\Shared\Application\Log\Config\SprykerLoggerConfig;
+use Spryker\Shared\Log\LogConstants;
+use Spryker\Shared\Propel\PropelConstants;
+use Spryker\Zed\PropelOrm\Business\Builder\ExtensionObjectBuilder;
+use Spryker\Zed\PropelOrm\Business\Builder\ExtensionQueryBuilder;
+use Spryker\Zed\PropelOrm\Business\Builder\ObjectBuilder;
+use Spryker\Zed\PropelOrm\Business\Builder\QueryBuilder;
+use Spryker\Shared\Kernel\KernelConstants;
+use Spryker\Shared\ErrorHandler\ErrorHandlerConstants;
+
+$connections = [
+    'mysql' => [
+        'adapter' => 'sqlite',
```

Comment:
what is the idea behind it?

for tests? only on local env?

Reply:
This is only for testing purposes and will never be used in production. I have no docker for this repository so I had to find another way to test against the DB.